jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -625,9 +634,74 @@ def _update_stretch_histogram(self, msg={}):
             # in case only the sample has changed but its length has not,
             # we'll force the traitlet to trigger a change
             hist_mark.send_state('sample')
-
         self._stretch_histogram_needs_update = False
 
+    @observe('stretch_vmin_value', 'stretch_vmax_value', 'layer_selected',
+             'stretch_function_value', 'stretch_hist_nbins',

```

Comment:
I think this is redundant with the `add_observe` above (although we might want a comment here as well that we are listening to `stretch_function_value`.

```suggestion
             'stretch_hist_nbins',

```

Reply:
Done,t hanks.